### PR TITLE
Fix coding standard in commons

### PR DESCRIPTION
**Pull request by BlopApple**

### CODE REVIEW

File: src/main/java/seedu/uninurse/commons/util/StringUtil.java [hunk 1]
```diff
@@ -10,18 +10,16 @@
...
+     * containsWordIgnoreCase("ABc def", "DEF") == true
+     * containsWordIgnoreCase("ABc def", "AB") == true //partial word match
      *
-     * @param sentence cannot be null
-     * @param word     cannot be null, cannot be empty, must be a single word
+     * @param sentence a string that cannot be null
+     * @param word     a string that cannot be null, cannot be empty, must be a single word
      */
     public static boolean containsWordIgnoreCase(String sentence, String word) {
         requireNonNull(sentence);
```
Comment (izzahaj) [inline src/main/java/seedu/uninurse/commons/util/StringUtil.java:25]: shouldn't this be `requireAllNonNull`?

File: src/main/java/seedu/uninurse/commons/util/StringUtil.java [hunk 2]
```diff
@@ -43,17 +41,16 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
...
+     * containsWordIgnoreCase("ABc def", "AB") == true //partial word match
+     * containsWordIgnoreCase("ABc def", "c d") == true //multiple word match
      *
-     * @param sentence cannot be null
-     * @param word     cannot be null
+     * @param sentence a string that cannot be null
+     * @param word     a string that cannot be null
      */
     public static boolean containsIgnoreCase(String sentence, String word) {
         requireNonNull(sentence);
```
Comment (izzahaj) [inline src/main/java/seedu/uninurse/commons/util/StringUtil.java:56]: same as above

File: src/main/java/seedu/uninurse/commons/util/StringUtil.java [hunk 3]
```diff
@@ -72,19 +69,19 @@ public static String getDetails(Throwable t) {
...
      * Will return false for any other non-null string input
      * e.g. empty string, "-1", "0", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
      *
-     * @throws NullPointerException if {@code s} is null.
+     * @throws NullPointerException if string is null.
      */
-    public static boolean isNonZeroUnsignedInteger(String s) {
-        requireNonNull(s);
+    public static boolean isNonZeroUnsignedInteger(String string) {
+        requireNonNull(string);
```
Comment (izzahaj) [inline src/main/java/seedu/uninurse/commons/util/StringUtil.java:80]: same as above

File: src/main/java/seedu/uninurse/commons/util/AppUtil.java [hunk 4]
```diff
@@ -9,19 +9,18 @@
  * A container for App specific utility functions
  */
 public class AppUtil {
-
     /**
-     * Gets an {@code Image} from the specified path.
+     * Gets an Image from the specified path.
      */
     public static Image getImage(String imagePath) {
         requireNonNull(imagePath);
```
Comment (izzahaj) [inline src/main/java/seedu/uninurse/commons/util/AppUtil.java:16]: same as below

**Review by izzahaj — APPROVED** [review]
I think you left out a few `requireNonNull`s that should be changed to `requireAllNonNull`. Not sure if this was intentional. Everything else looks ok so I'll approve.

**Comment by BlopApple** [comment]
The idea is that the other commons classes would not have dependencies on `requireAllNonNull` from `CollectionUtil`.

**Pull request merged**